src/presets/yfm.ts

```diff
@@ -64,5 +67,6 @@ export const YfmPreset: ExtensionAuto<YfmPresetOptions> = (builder, opts) => {
         .use(YfmFile, opts.yfmFile ?? {})
         .use(YfmHeading, opts.yfmHeading ?? {})
         .use(YfmTable, opts.yfmTable ?? {})
-        .use(YfmTabs);
+        .use(YfmTabs)
+        .use(YfmHtmlBlock, opts.yfmHtmlBlock ?? {});
```

Comment:
YfmHtmlBlock should be optional and plug-in via extaExtesion

Reply:
fixed